roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Install NGINX
+  package:
+    name: nginx
+    state: present
+  become: yes
```

Comment:
maybe put the become tags in the role in the playbook.yml file? Like the common role.

Reply:
True. 
done